src/lcell.rs

```diff
@@ -146,6 +146,17 @@ impl<'id, T> LCell<'id, T> {
             value: UnsafeCell::new(value),
         }
     }
+
+    /// Get a pointer into the cell
+    /// 
+    /// # Safety
+    /// 
+    /// It is only safe to write to this pointer while the cell is
+    /// not being held by a `ro` lock.
```

Comment:
There must not be a reference derived from 'rw' active either. If there was, then there would be two mutable references to the same memory.

Reply:
Yes, thanks for the catch